backend/src/database/migrations/20220708084442-create-user.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;');
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      email: {
+        type: Sequelize.CITEXT,
+        allowNull: false,
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      form: {
+        type: Sequelize.JSONB,
+        allowNull: true,
+      },
+      raspored: {
+        type: Sequelize.JSONB,
+        allowNull: true,
+      },
```

Comment:
Please remove these two fields for now and add equivalents when you deem appropriate.

Reply:
Fields have been removed. Will do.